### DISCUSSION TITLE
iOS App Store does not allow Website to App.

### DISCUSSION THREAD

**Discussion by mantresh-pvt** [Spyxpo/web-to-app#30] [Q&A] [answered]
Apple does not allow Website to App uploaded on there App Store(App Store Guidelines), then why SWAB is going to support iOS App Builder?

**Comment by mantreshkhurana** [comment, marked as answer]
We know that Apple does not allow Website to App on there App Store but we still offer iOS Website to App conversion because we want SWAB to convert website to app for any platform.

Soon Apple will allow users to side load apps.

[Source](https://www.bloomberg.com/news/articles/2022-12-13/will-apple-allow-users-to-install-third-party-app-stores-sideload-in-europe)
